apidoc/Titanium/UI/TableView.yml

```diff
@@ -695,6 +695,40 @@ events:
         type: Boolean
     since: '3.0.0'
 
+  - name: editaction
```

Comment:
maybe `editAction` as it is called `editActions` below?

Reply:
Event names were historically lowercase-only in Titanium (with a few rare exceptions) and the listview-one also had this naming, so I took over this pattern